codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -1387,7 +1400,7 @@ getDependencyIdsForDependent dependent@(C.Reference.Id oid0 _) =
         FROM dependents_index
         WHERE dependency_builtin IS NULL
           AND dependent_object_id = ?
-          AND dependen_component_index = ?
+          AND dependent_component_index = ?
```

Comment:
I would've expected to see errors or something from this coming up; but it seems this combinator is actually dead code and isn't ever called 🤔

Reply:
Yikes, ok.